src/azure-cli/azure/cli/command_modules/synapse/tests/latest/test_synapse_scenario.py

```diff
@@ -346,6 +347,57 @@ def test_sql_pool_audit_policy(self):
                      self.check('state', 'Enabled')
                  ])
 
+    @record_only()
+    def test_sql_aad_admin(self):
+        self.kwargs.update({
+            'location': 'eastus',
+            'workspace': 'testsynapseworkspace',
+            'rg': 'rg',
+            'user-name': 'fakeuser',
+            'object-id': '00000000-0000-0000-0000-000000000000',
+            'user-email': 'fakeuser@fakedomain.com'
+        })
+        # Test create cmdlet
+        self.cmd('az synapse sql ad-admin create --workspace-name {workspace} --resource-group {rg} '
+                 '--display-name {user-name} --object-id {object-id}',
+                 checks=[
+                     self.check('login', self.kwargs['user-name'])
+                 ])
+
+        # Test show cmdlet
+        self.cmd('az synapse sql ad-admin show --workspace-name {workspace} --resource-group {rg}',
+                 checks=[
+                     self.check('login', self.kwargs['user-name']),
+                     self.check('name', 'activeDirectory')
+                 ])
+
+        # Test update cmdlet
+        self.cmd('az synapse sql ad-admin update --workspace-name {workspace} --resource-group {rg} '
+                 '--display-name {user-email}',
+                 checks=[
+                     self.check('login', self.kwargs['user-email'])
+                 ])
+        # Test delete cmdlet
+        self.cmd('az synapse sql ad-admin delete --workspace-name {workspace} --resource-group {rg} -y')
+        self.cmd('az synapse sql ad-admin show --workspace-name {workspace} --resource-group {rg}', expect_failure=True)
+
+    @record_only()
```

Comment:
why use record only here?

Reply:
For this test we need an existing aad object id which is very confidential.